cluster/addons/metadata-agent/stackdriver/metadata-agent.yaml

```diff
@@ -45,27 +29,22 @@ spec:
     spec:
       serviceAccountName: metadata-agent
       containers:
-      - image: gcr.io/stackdriver-agents/stackdriver-metadata-agent:0.2-0.0.19-1
+      - image: gcr.io/stackdriver-agents/stackdriver-metadata-agent:0.2-0.0.21-1
         imagePullPolicy: IfNotPresent
         name: metadata-agent
         livenessProbe:
-          exec:
-            command:
-              - /bin/bash
-              - -c
-              - |
-                if [[ -f /var/run/metadata-agent/health/unhealthy ]]; then
-                  exit 1;
-                fi
-          periodSeconds: 10
+          httpGet:
```

Comment:
Same here. Is this health probe change on purpose? Or something got in by accident when addressing the rebasing conflicts?

Reply:
The new health checks use http-based health checks to a handler that the API server exposes. Is this your concern? I feel that I might be missing something when you point out a rebase conflict.

Reply:
Ah. These changes were probably there before the rebase then. I was not sure of that and wanna double check.

Looks good.